src/Query.php

```diff
@@ -203,25 +203,31 @@ public function buildCursor($db = null)
 
     /**
      * Fetches rows from the given Mongo cursor.
-     * @param \MongoDB\Driver\Cursor $cursor Mongo cursor instance to fetch data from.
+     * @param yii\mongodb\Connection $db the MongoDB connection used to fetch rows.
      * @param bool $all whether to fetch all rows or only first one.
      * @param string|callable $indexBy the column name or PHP callback,
      * by which the query results should be indexed by.
      * @throws Exception on failure.
      * @return array|bool result.
      */
-    protected function fetchRows($cursor, $all = true, $indexBy = null)
+    protected function fetchRows($db, $all = true, $indexBy = null)
```

Comment:
Looks like change of method signature. Is there a way to avoid it?

Reply:
i think not exists any way because we can not access to db connection in `fetchRows()`.

Reply:
Then it can be included in major release, such as 2.2.0, only.